### ISSUE TITLE
Sharing InternList 🙂 

### ISSUE THREAD

**Issue by BuckyMaler** [pittcsc/Summer2023-Internships#4675]
### Issue/Question

Hi! 👋 

Hats off to the maintainers and contributors of this repo. It's a fantastic resource.

I built [InternList](https://intern-list.vercel.app) as a small side project to experiment with and introduce some AI tooling into my dev workflow. While the main reason for the project was to get exposure to AI tooling, I also tried to build something useful. If you also think it's useful and would like to list it beside SWE List (pretty cool by the way) in your `README.md` that'd be super cool. I consider it an MVP right now. If it proves to be useful, I'll continue to make enhancements.

**Comment by BuckyMaler** [comment]
@GaryZhous 👋 I was hoping to get a maintainers 👀 on this. [InternList](https://intern-list.vercel.app) currently uses this repo's data exclusively. InternList is searchable/filterable by company, position, location, term, and H-1B sponsorship support. The UI/UX is pretty good and I plan to make enhancements as it gains traction. Could you consider adding it to the readme please? 🙏 I'd be happy to provide an attribution link from the app back to this repo.

**Comment by GaryZhous** [comment]
Sorry for the late response. This is a great list thanks for the effort, but I will have to ask Michael about it. I have assigned him to this issue and hopefully he can get back to you asap!

**Comment by yan-michael** [comment]
Shoot me an email at michael at simplify dot jobs! Happy to chat there!

**Comment by BuckyMaler** [comment]
> Shoot me an email at michael at simplify dot jobs! Happy to chat there!

@yan-michael I sent you a couple of emails (subject: Bucky Maler - Intern List) over the past couple weeks, but with the holidays they may have gotten lost in the mix. I'm not looking to spam you, as I'm sure you're busy, so I'll leave this as my final message.

I've since added the new grad positions to the website, renamed Intern List to Good First Job, and added attribution with links back to the repos. Would it be ok if I opened PRs adding [Good First Job](https://www.goodfirstjob.com) to the READMEs?

Side note, I think Good First Job and SWE List would integrate well together. I spoke with Matt Reed before finding out that he no longer owns SWE List and he thought the idea sounded cool.
